worker/task_test.go

```diff
@@ -53,6 +57,7 @@ var claim = &taskClaim{
 			ClientID:    "abc",
 			Certificate: "",
 		},
+		TakenUntil: tcclient.Time(time.Now().Add(time.Minute * 5)),
```

Comment:
TakenUntil is just a time stamp https://docs.taskcluster.net/reference/platform/queue/api-docs#reclaimTask